packages/shared/src/hooks/squads/useSourceModerationList.ts

```diff
@@ -98,14 +103,34 @@ export const useSourceModerationList = ({
     user,
     squad.id,
   );
-  const squadQueryKey = generateQueryKey(RequestKey.Squad, user, squad.handle);
```

Comment:
Might be wrong, but invalidating / setting squad data didn't seem to be necessary. Posts appear in their respective squads when approved.